### DISCUSSION TITLE
Cannot Login to https://openlogin.com

### DISCUSSION THREAD

**Discussion by metasoul111** [Web3Auth/Web3Auth#1181] [Help]
Starting around 7 am CST this morning our users started experiencing this.

This issue is still persisting and happening quite often right now. 

<img width="1205" alt="Screenshot 2023-01-06 at 11 40 55 PM" src="https://user-images.githubusercontent.com/89988259/211065619-1cbb40aa-38e6-4ac7-ac4b-4b91b6df3dd0.png">

**Comment by metasoul111** [comment]
@chaitanyapotti

**Comment by metasoul111** [comment]
Bumping this, can anyone help @chaitanyapotti @tainguyentt

**Comment by shahbaz17** [comment]
Hey @metasoul111 

Which login method are you using to log in to https://app.openlogin.com?

**Reply by metasoul111** [reply]
@shahbaz17 we are using email method via API.
